### ISSUE TITLE
AmiGO (production) currently has a bad load due to incorrect configuration and unknown owltools issues

### ISSUE THREAD

**Issue by kltm** [geneontology/amigo#262]
As a stopgap for users, make amigo2.berkeleybop.org the safe good 2.3.x load.

AmiGO "example" configs, jenkins jobs, and downstream clients (Noctua, etc.) need to reflect that tomodachi is the dev load on master and nakama is the 2.3.x "production" load, as well as the new "amigo2".

This will also make it easier to address emergencies such as right now: http://jira.geneontology.org/browse/GO-1011.

**Comment by kltm** [comment]
amigo2.berkeleybop.org now passes back to nakama.berkeleybop.org.

**Comment by kltm** [comment]
Tagging on @cmungall, @stuartmiyasato, and @kkarra.

**Comment by kltm** [comment]
When closed, re-run install on nakama to internal links understand that it is the new amigo2 @ berkeley.

**Comment by cmungall** [comment]
broken:

http://golr.geneontology.org/solr/select?q=id:%22GO:0090069%22

```
<str name="regulates_transitivity_graph_json">{"nodes":[{"id":"GO:0008150","lbl":"biological_process"},{"id":"GO:0065007","lbl":"biological regulation"},{"id":"GO:0044087","lbl":"regulation of cellular component biogenesis"},{"id":"GO:0090069","lbl":"regulation of ribosome biogenesis"},{"id":"GO:0050789","lbl":"regulation of biological process"}],"edges":[{"sub":"GO:0090069","obj":"GO:0050789","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0065007","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0008150","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0008150","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0044087","pred":"is_a"}]}

<arr name="regulates_closure"><str>GO:0065007</str><str>GO:0008150</str><str>GO:0090069</str><str>GO:0044087</str><str>GO:0050789</str></arr><arr name="regulates_closure_label"><str>biological regulation</str><str>biological_process</str><str>regulation of ribosome biogenesis</str><str>regulation of cellular component biogenesis</str><str>regulation of biological process</str></arr>

```

http://toaster.lbl.gov:9000/solr/select?q=id:%22GO:0090069%22

```
<str name="regulates_transitivity_graph_json">{"nodes":[{"id":"GO:0071840","lbl":"cellular component organization or biogenesis"},{"id":"GO:0044087","lbl":"regulation of cellular component biogenesis"},{"id":"GO:0044085","lbl":"cellular component biogenesis"},{"id":"GO:0042254","lbl":"ribosome biogenesis"},{"id":"GO:0065007","lbl":"biological regulation"},{"id":"GO:0022613","lbl":"ribonucleoprotein complex biogenesis"},{"id":"GO:0050789","lbl":"regulation of biological process"},{"id":"GO:0090069","lbl":"regulation of ribosome biogenesis"},{"id":"GO:0008150","lbl":"biological_process"}],"edges":[{"sub":"GO:0090069","obj":"GO:0008150","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0042254","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0044085","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0071840","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0022613","pred":"regulates"},{"sub":"GO:0090069","obj":"GO:0065007","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0050789","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0044087","pred":"is_a"},{"sub":"GO:0090069","obj":"GO:0008150","pred":"is_a"}]}

<arr name="regulates_closure"><str>GO:0044087</str><str>GO:0008150</str><str>GO:0090069</str><str>GO:0042254</str><str>GO:0044085</str><str>GO:0050789</str><str>GO:0065007</str><str>GO:0022613</str><str>GO:0071840</str></arr><arr name="regulates_closure_label"><str>regulation of cellular component biogenesis</str><str>biological_process</str><str>regulation of ribosome biogenesis</str><str>ribosome biogenesis</str><str>cellular component biogenesis</str><str>regulation of biological process</str><str>biological regulation</str><str>ribonucleoprotein complex biogenesis</str><str>cellular component organization or biogenesis</str></arr>
```

**Comment by kltm** [comment]
There are two parts to this now: 1) triaging amigo.geneontology.org in the short term to point to a "good" source until there can be a reload and 2) figuring out why the load went bad in the first place.

**Comment by kltm** [comment]
Triaging is now complete, with amigo.geneontology.org kicking back to good amigo2.berkeleybop.org. We're not going to worry about golr clients at this point (who should use safe toaster.lbl.gov:9000/solr/ in the interim if necessary).

**Comment by kltm** [comment]
After apparent firewall block at production (due to proxypass?), A record for amigo.geneontology.org now pointing to groovin.

**Comment by kltm** [comment]
For no. 2, the issue was identified as owltools importer behaviour; the amigo-specific fixes are outlined on #263.

**Comment by kltm** [comment]
Will leave work on production issues around the reloading and DNS to other items and discussions.
